### PR TITLE
Aw add memory to host info

### PR DESCRIPTION
**Pull request by andywong418**

### CODE REVIEW

File: src/controllers/info.ts [hunk 1]
```diff
@@ -15,17 +15,22 @@ export default function (server: Hapi.Server, deps: Injector) {
...
     }
   }
 
   async function infoHandler (request: Hapi.Request, h: Hapi.ResponseToolkit) {
     const fullMem = podManager.getMemoryUsed() * (2 ** 20) / os.totalmem() >= config.maxMemoryFraction
+    const serverMemory = freeMem()
     const infoResp: HostInfo = {
       fullMem,
       acceptingUploads: !fullMem,
+      serverMemory,
```
Comment (sharafian) [inline src/controllers/info.ts]: I didn't know that this was just the free memory; maybe we could call it `serverFreeMemory` instead

**Comment by steffenmandrup** [comment]
Shouldnt fullMem showcase full memory and not a bool? Then /memory can be removed

**Comment by sharafian** [comment]
I agree that the name is confusing, but it's an indicator whether the memory is full or not. We've got the variable "acceptingUploads" now, which is clearer.

We don't want to change existing fields or remove endpoints because then it's a breaking change

**Comment by steffenmandrup** [comment]
Okay :-)

**Pull request merged**